server/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -63,7 +72,7 @@
     private static final ParseField SLOP_FIELD = new ParseField("slop");
     private static final ParseField ZERO_TERMS_QUERY_FIELD = new ParseField("zero_terms_query");
     private static final ParseField LENIENT_FIELD = new ParseField("lenient");
-    private static final ParseField CUTOFF_FREQUENCY_FIELD = new ParseField("cutoff_frequency");
+    private static final ParseField CUTOFF_FREQUENCY_FIELD = new ParseField("cutoff_frequency", "cutoff_frequency");
```

Comment:
I think you can replace this with  `new ParseField("cutoff_frequency") .withAllDeprecated("CUTOFF_FREQUENCY_DEPRECATION_MSG")`